doc/ref/user_pref_list.xml

```diff
@@ -0,0 +1,293 @@
+
+<P/>
+
+Admissible values:
+<C>1</C>,
+<C>2</C>,
+<C>3</C>,
+<C>4</C>.
+
+<P/>
+
+Default: <C>1</C>.
+</Item>
+<Mark>
+<Index Key='MaxBitsIntView'><C>MaxBitsIntView</C></Index>
+<C>MaxBitsIntView</C>
+</Mark>
+<Item>
+Maximal bit length of integers to <C>View</C> unabbreviated. Default is about
+<M>30</M> lines of a <M>80</M> character wide terminal. Set this to <C>0</C>
+to avoid abbreviated ints.
+
+<P/>
+
+Default: <C>8000</C>.
+</Item>
+<Mark>
+<Index Key='PartialPermDisplayLimit'><C>PartialPermDisplayLimit</C></Index>
+<Index Key='NotationForPartialPerms'><C>NotationForPartialPerms</C></Index>
+<C>PartialPermDisplayLimit</C>,
+<C>NotationForPartialPerms</C>
+</Mark>
+<Item>
+options for the display of partial perms
+
+<P/>
+
+Defaults: <C>[ 100, "component" ]</C>.
+</Item>
+<Mark>
+<Index Key='TransformationDisplayLimit'><C>TransformationDisplayLimit</C></Index>
+<Index Key='NotationForTransformations'><C>NotationForTransformations</C></Index>
+<C>TransformationDisplayLimit</C>,
+<C>NotationForTransformations</C>
+</Mark>
+<Item>
+options for the display of transformations
+
+<P/>
+
+Defaults: <C>[ 100, "input" ]</C>.
+</Item>
+<Mark>
+<Index Key='PackagesToIgnore'><C>PackagesToIgnore</C></Index>
+<C>PackagesToIgnore</C>
+</Mark>
+<Item>
+These packages are not regarded as available. This doesn't work for packages
+which are needed by the &GAP; library, or which are already loaded in a
+workspace.
+
+<P/>
+
+Default: <C>""</C>.
+</Item>
+<Mark>
+<Index Key='PackagesToLoad'><C>PackagesToLoad</C></Index>
+<C>PackagesToLoad</C>
+</Mark>
+<Item>
+A list of names of packages which should be loaded during startup. For
+backwards compatibility, the default lists most of packages that were
+autoloaded in &GAP; 4.4 (add or remove packages as you like).
+
+<P/>
+
+Default: <C>[ "autpgrp", "alnuth", "crisp", "ctbllib", "factint", "fga", "irredsol", "laguna", "polenta", "polycyclic", "resclasses", "sophus", "tomlib" ]</C>.
+</Item>
+<Mark>
+<Index Key='Pager'><C>Pager</C></Index>
+<Index Key='PagerOptions'><C>PagerOptions</C></Index>
+<C>Pager</C>,
+<C>PagerOptions</C>
+</Mark>
+<Item>
+For displaying help pages on screen and other things &GAP; has a rudimentary
+builtin pager. We recommend using a more sophisticated external program. For
+example, when you have the program <C>less</C> on your computer we recommend:
+<P/>
+<C>Pager := "less";</C>
+<P/>
+<C>PagerOptions := ["-f", "-r", "-a", "-i", "-M", "-j2"];</C>
+<P/>
+If you want to use <C>more</C>, we suggest to use the <C>-f</C> option. If you
+want to use the pager defined in your environment then leave the <C>Pager</C>
+and <C>PagerOptions</C> preferences empty.
+
+<P/>
+
+The defaults are computed at runtime.
+</Item>
+<Mark>
+<Index Key='ReadObsolete'><C>ReadObsolete</C></Index>
+<C>ReadObsolete</C>
+</Mark>
+<Item>
+May be useful to say <K>false</K> here to check if you are using commands
+which may vanish in a future version of &GAP;
```

Comment:
Ugh, this text really should be improved. But of course that's out of scope for this PR!

Reply:
Yes, some of the "description" texts have not been written under the assumption that they will appear in the manual.